tests/cxx/quoted_includes_first.h

```diff
@@ -13,21 +13,21 @@
 #include <memory>
 #include "subdir/indirect_subdir.h"
 
-std::unique_ptr<IndirectSubDirClass> CreateIndirectSubDirClass();
+std::unique_ptr<int> CreateIndirectSubDirClass();
+
+IndirectSubDirClass global;
```

Comment:
Nit: it should better be marked as `inline` to make the code closer to correct C++.

Reply:
Thanks, made it `inline const`, even, since the source-file function copies it.